.github/workflows/release.yaml

```diff
@@ -5,46 +5,50 @@ on:
     - master
   schedule:
   - cron: "0 */8 * * *"
+  workflow_dispatch: {}
 jobs:
   release:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@v1
     - name: Prepare checking latest kubectl version
-      run: docker build . -t line/kubectl-kustomize:kubectl-version --target latest-kubectl-version
+      env:
+        REPO_OWNER: ${{github.repository_owner}}
+      run: docker build . -t ${REPO_OWNER}/kubectl-kustomize:kubectl-version --target latest-kubectl-version
     - name: Check latest kubectl version
       id: kubectl
+      env:
+        REPO_OWNER: ${{github.repository_owner}}
       run: |
-        if version=$(docker run line/kubectl-kustomize:kubectl-version); then
+        if version=$(docker run ${REPO_OWNER}/kubectl-kustomize:kubectl-version); then
           echo ::set-output name=version::$version
         else
           exit 1
         fi
     - name: Prepare checking latest kustomize release
+      env:
+        REPO_OWNER: ${{github.repository_owner}}
       run: |
-        docker build . -t line/kubectl-kustomize:kustomize-version --target latest-kustomize-version
-        docker build . -t line/kubectl-kustomize:kustomize-url --target latest-kustomize-download-url
+        docker build . -t ${REPO_OWNER}/kubectl-kustomize:kustomize-version --target latest-kustomize-version
```

Comment:
I switched from looking up the kustomize download url to the kustomize version. The download URL will differ from one architecture to another, but all of them follow a predictable template, so the version is all we need to figure out the url.